comp150-agile/essay.tex

```diff
@@ -0,0 +1,65 @@
+% Please do not change the document class
+\documentclass{scrartcl}
+
+% Please do not change these packages
+\usepackage[hidelinks]{hyperref}
+\usepackage[none]{hyphenat}
+\usepackage{setspace}
+\doublespace
+
+% You may add additional packages here
+\usepackage{amsmath}
+
+% Please include a clear, concise, and descriptive title
+\title{Do Daily Scrums Meetings Improve Interpersonal Relations Within a Game Development Team?}
+
+% Please do not change the subtitle
+\subtitle{COMP150 - Agile Development Practice}
+
+% Please put your student number in the author field
+\author{1607804}
+
+\begin{document}
+
+\maketitle
+
+
+\section{Introduction}
+There is limited research on the topic of Daily scrum meetings and whether they improve the interpersonal relations of the team. However there is a lot of theory around team effectiveness across many work disciplines which can be extrapolated to fit the principles of Agile Software Development \cite{Agile}. 

```

Comment:
To help *motivate* this research, you could refer some some [opinion pieces](http://dl.acm.org/citation.cfm?id=957730&CFID=697395244&CFTOKEN=53777058) in professional magazines that criticise stand-up meetings. This would better situation and justify your musings and explorations.